.github/workflows/e2e-infra-destroy.yml

```diff
@@ -0,0 +1,44 @@
+# PROCESS
+#
+# 1. Destroy the core stack using AWS CDK.
+# 2. Destroy the AOT stack using AWS CDK.
+
+# USAGE
+#
+# This workflow is used by the E2E Tests workflow to destroy the infrastructure after tests are completed.
+
+name: Destroy e2e CDK Stacks
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  run-tests:
+    uses: ./.github/workflows/e2e-tests.yml
+
+  destroy-stacks:
+    runs-on: aws-powertools_ubuntu-latest_8-core
+    needs: [run-tests]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
```

Comment:
As a best practices, please put the sha256 that matches this version. Dependabot will bump it.